material-overrides/assets/stylesheets/tanssi.css

```diff
@@ -1342,4 +1342,57 @@ h2.subtitle.mobile-subtitle {
   border-radius: 1em;
   vertical-align: baseline;
   box-shadow: none;
+}
+
+/* Cards Styling */ 
+.md-typeset .grid.cards>ul>li {
+  border: 1px solid #FFFAFA;
+  background-color: #3D3D3D;
+}
+.md-typeset .grid.cards>ul>li:hover {
+  border: 1px solid #189A9A;
+  background-color: #1B1B1B;
```

Comment:
All colors need to be set as variables (for example, like how there is `--md-default-fg-color`). That way it'll look good on light and dark mode.